cyclops-ui/src/components/pages/TemplateStore/TemplateStore.tsx

```diff
@@ -182,8 +193,18 @@ const TemplateStore = () => {
           </Button>
         </Col>
       </Row>
+      <Row gutter={[40, 0]}>
+        <Col span={18}>
+          <Input
+            placeholder={"Search templates"}
+            style={{ width: "30%" }}
+            onChange={handleSearch}
+          ></Input>
+        </Col>
+      </Row>
+      <Divider orientationMargin="0" />
       <Col span={24} style={{ overflowX: "auto" }}>
-        <Table dataSource={templates}>
+        <Table dataSource={filteredTemplates}>
```

Comment:
Can we remove the divider and add some padding to the searchbar?
```html
      <Row gutter={[40, 0]} style={{paddingBottom: "16px"}}>
        <Col span={18}>
          <Input
            placeholder={"Search templates"}
            style={{ width: "30%" }}
            onChange={handleSearch}
          ></Input>
        </Col>
      </Row>
```

Reply:
Ya we can do that, check if this much spacing ok

![image](https://github.com/user-attachments/assets/eda097c9-8708-4594-8a6a-13d82fbe0a2a)

Reply:
@kom-senapati yup, looks good